### ISSUE TITLE
Pozyskanie wszystkich parametrów dla każdej kategorii

### ISSUE THREAD

**Issue by TuanDangIT** [allegro/allegro-api#9031]
Dzień dobry.

Czy jest jakaś inna możliwość pozyskania wszystkich parametrów dla każdej kategorii. Chodzi o to, że stosując teraz po kolei endpoint /sale/categories/{categoryId}/parameters dla każdej kategorii jest po 1 czasochłonne, po drugie może skutkować to blokadą z strony waszego API, po wysłaniu za dużo zapytań HTTP.

Pozdrawiam,
Tuan Dang.

**Comment by AureliuszBrussy** [comment]
Nie, nie ma innej możliwości, [GET /sale/categories/{categoryId}/parameters](https://developer.allegro.pl/documentation#operation/getFlatParametersUsingGET_2) to jedyny endpoint, który umożliwia pobranie komplentych parametrów.

**Comment by TuanDangIT** [comment]
Też rozumiem, że nigdzie na macie zapisane po kolei kategorie + parametry?

**Comment by AureliuszBrussy** [comment]
Nie, tak jak wspomniałem, dla pobrania dostępnych parametrów to jedyny endpoint. Nie mamy tutaj dodatkowych limitów na endpointcie, poza limitem ogólnym publicznego API - 9000 zapytań na minutę.

**Comment by TuanDangIT** [comment]
Rozumiem, że na konto 9000 zapytań/minutę czy na IP?

**Comment by AureliuszBrussy** [comment]
9000 req/min na Client ID.

**Comment by TuanDangIT** [comment]
Mam pytanie do sandbox Allegro. Czy można w jakiś sposób testować generowanie etykiet dostaw? Nie można skończyć procesu zwrotu przez etykietę.![image](https://github.com/allegro/allegro-api/assets/128043070/52b95a87-4f02-4246-94fc-2ca968d20097)

**Comment by MartaNowaczyk** [comment]
Zwrot jest już pewnie zarejestrowany. Możesz też podczas procesu wybrać przesyłkę własną. Jeżeli zauważyłeś, że zwrot się nie wygenerował zgłoś to proszę przez [formularz kontaktowy]( https://allegro.pl/pomoc/kontakt?kategoria=e2e1f990-74d6-4bea-9b5c-a408bde7983d&subjectId=da43023c-bc80-44ec-9a6d-d43df46b42a6).

**Comment by TuanDangIT** [comment]
Jest procesowanie zwrotu przez Allegro sandbox a nie na zwykłym Allegro.

**Comment by TuanDangIT** [comment]
Jasne rozumiem. To wyślę formularz w takim razie.
